src/System.Drawing.Common/src/System/Drawing/NativeStructs.Unix.cs

```diff
@@ -34,7 +34,7 @@
 
 namespace System.Drawing
 {
-    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
```

Comment:
Is this always used with `W` variants?

Reply:
Yep. We never use the A variant on Unix as far as I can tell.